www/index.php

```diff
@@ -2,7 +2,9 @@
 
 declare(strict_types=1);
 
-$container = require __DIR__ . '/../app/bootstrap.php';
+require __DIR__ . '/../vendor/autoload.php';
 
-$container->getByType(Nette\Application\Application::class)
+App\Application::boot()
+	->createContainer()
+	->getByType(Nette\Application\Application::class)
```

Comment:
using two different classes named `Application` in once file si a terrible idea.

Reply:
Suggest better name.

Reply:
Kernel?